### ISSUE TITLE
STT is not listening when we have background noises.

### ISSUE THREAD

**Issue by kapilkarda** [deepgram/deepgram-js-sdk#318]
## What is the current behavior?
It is working fine when their is no background noise like music, traffic or gossips by other.

> What's happening that seems wrong?
It should avoid the background noise and it should work with background noise as well.

## Steps to reproduce

> To make it faster to diagnose the root problem. Tell us how can we reproduce the bug.

I am running this scrip https://github.com/twilio-labs/call-gpt

## Expected behavior

> What would you expect to happen when following the steps above?

## Please tell us about your environment

i am running it on aws ubuntu instance.

> We want to make sure the problem isn't specific to your operating system or programming language.
  
- **Operating System/Version:** Windows 10
- **Language:** [all | TypeScript | Python | PHP | etc]
- **Browser:** Chrome

## Other information

> Anything else we should know? (e.g. detailed explanation, stack-traces, related issues, suggestions how to fix, links for us to have context, eg. stack overflow, codepen, etc)

**Comment by lukeocodes** [comment]
Did you try to change the model to a phonecall model?

**Comment by kapilkarda** [comment]
Yes, I did thats not working as well and I want for hindi language as well but it didn’t work.

**Comment by lukeocodes** [comment]
Then this appears to be a model related issue. You'll get faster support on the [official discord](https://discord.gg/deepgram)
